rispy/parser.py

```diff
@@ -187,6 +187,7 @@ def load(
     mapping: Optional[Dict] = None,
     implementation: RisImplementation = RisImplementation.BASE,
     strict: bool = True,
+    **kw,
```

Comment:
I don't think this is the best way to implement this. I think it should be the second position (the other arguments are options of the parser) and not with a variable-length argument. 

```
def load(    
    file: Union[TextIO, Path],
    encoding: str,
    mapping: Optional[Dict] = None,
    implementation: RisImplementation = RisImplementation.BASE,
    strict: bool = True,
    **kw,
```

Reply:
I think it should also be added to the docstring.